gst/nnstreamer/elements/gsttensor_trainer.c

```diff
@@ -195,6 +220,43 @@ gst_tensor_trainer_class_init (GstTensorTrainerClass * klass)
           G_PARAM_READWRITE | GST_PARAM_MUTABLE_READY |
           G_PARAM_STATIC_STRINGS));
 
+  g_object_class_install_property (gobject_class, PROP_PUSH_OUTPUT,
+      g_param_spec_boolean ("push-output", "Push output tensor",
+          "Add output tensors to output GstBuffer", FALSE,
+          G_PARAM_READWRITE | GST_PARAM_MUTABLE_READY |
+          G_PARAM_STATIC_STRINGS));
+
+  g_object_class_install_property (gobject_class, PROP_NUM_INPUTS,
+      g_param_spec_uint ("num-inputs", "Number of inputs",
+          "An input in a tensor can have one or more features data,"
+          "set how many inputs are received", 0, NNS_TENSOR_SIZE_LIMIT, 1,
+          G_PARAM_READWRITE | GST_PARAM_MUTABLE_READY |
+          G_PARAM_STATIC_STRINGS));
+
+  g_object_class_install_property (gobject_class, PROP_NUM_LABELS,
+      g_param_spec_uint ("num-labels", "Number of labels",
+          "A label in a tensor can have one or more classes data,"
+          "set how many labes are recevied", 0, NNS_TENSOR_SIZE_LIMIT, 1,
+          G_PARAM_READWRITE | GST_PARAM_MUTABLE_READY |
+          G_PARAM_STATIC_STRINGS));
+
+  g_object_class_install_property (gobject_class, PROP_NUM_TRAINING_SAMPLES,
+      g_param_spec_uint ("num-training-samples", "Number of training samples",
+          "A sample can consist of muliple inputs and labels in tensors of a gstbuffer"
+          ", set how many samples are taken for training model", 0, G_MAXUINT,
+          1,
+          G_PARAM_READWRITE | GST_PARAM_MUTABLE_READY |
+          G_PARAM_STATIC_STRINGS));
+
+  g_object_class_install_property (gobject_class, PROP_NUM_VALIDATION_SAMPLES,
+      g_param_spec_uint ("num-validation-samples",
+          "Number of validation samples",
+          "A sample can consist of muliple inputs and labels in tensors of a gstbuffer"
```

Comment:
"A sample can consist of muliple inputs and labels in tensors of a gstbuffer"
-> "A sample can consist of **muliple** inputs and labels in tensors of a gstbuffer"

Reply:
Thank you.

Reply:
fixed, thank you.